grunt_tasks/webpack.config.js

```diff
@@ -92,11 +130,13 @@ module.exports = {
             // JSON files
             {
                 test: /\.json$/,
+                include: loaderPaths,
```

Comment:
Is it necessary to strictly enumerate the include dirs? Since the core and each plugin are all built with separate webpack configs, it seems like we might be able to get away without doing this at all